docs/data/material/components/snackbars/snackbars.md

```diff
@@ -9,109 +9,88 @@ waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
 
-[Grow](/material-ui/transitions/#grow) is the default transition but you can use a different one.
+### Use with Alerts
 
-{{"demo": "TransitionsSnackbar.js"}}
+Use an alert inside a snackbar for messages that communicate a certain severity.
 
-### Control Slide direction
+{{"demo": "CustomizedSnackbars.js"}}
 
-You can change the direction of the [Slide](/material-ui/transitions/#slide) transition.
+### Use with Floating Action Buttons
 
-Example of making the slide transition to the left:
+If you're using a [Floating Action Button](/material-ui/react-floating-action-button/) on mobile, Material Design recommends positioning any snackbar right above it.
+The demo below shows you how to do that.
 
-```jsx
-import Slide from '@mui/material/Slide';
+{{"demo": "FabIntegrationSnackbar.js", "iframe": true, "maxWidth": 400}}
 
-function TransitionLeft(props) {
-  return <Slide {...props} direction="left" />;
-}
+## Common examples
 
-export default function MyComponent() {
-  return <Snackbar TransitionComponent={TransitionLeft} />;
-}
-```
-
-Other examples:
+### Consecutive Snackbars
 
-{{"demo": "DirectionSnackbar.js"}}
+This demo shows how to show multiple snackbars without stacking them by using a consecutive animation.
 
-## Complementary projects
+{{"demo": "ConsecutiveSnackbars.js"}}
 
-For more advanced use cases you might be able to take advantage of:
+## Third-party intergrations
 
 ### notistack
 
 ![stars](https://img.shields.io/github/stars/iamhosseindhv/notistack.svg?style=social&label=Star)
 ![npm downloads](https://img.shields.io/npm/dm/notistack.svg)
 
-This example demonstrates how to use [notistack](https://github.com/iamhosseindhv/notistack).
-notistack has an **imperative API** that makes it easy to display snackbars, without having to handle their open/close state.
-It also enables you to **stack** them on top of one another (although this is discouraged by the Material Design guidelines).
+With an imperative API, [notistack](https://github.com/iamhosseindhv/notistack) allows you to vertically stack multiple Snackbars without having to handle their open and close states.
+Even though this is discouraged in the Material Design guidelines, it is still a common pattern.
 
 {{"demo": "IntegrationNotistack.js", "defaultCodeOpen": false}}
 
 ## Accessibility
 
-(WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
+Here are a few tips to make sure you have an accessible Snackbar component:
 
-By default, the snackbar won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended to give the user [sufficient time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html) to respond.
+- Based on the [WAI-ARIA guidelines](https://www.w3.org/TR/wai-aria-1.1/#alert), the Snackbar doesn't auto-hide by default.
+  If you use the `autoHideDuration` prop, make sure to [provide sufficient time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html) for the user to process the information displayed on it.
```

Comment:
this page doesn't mention this prop elsewhere, does it? Maybe we should add a section and a demo to show it off? But then again, it seems like we should probably discourage devs from actually using it (according to WAI-ARIA), so, maybe not... ?

Reply:
Uhm... good question. Interestingly, though, and not sure if I misunderstood it, I didn't feel like the Aria link there mentions precisely what we're recommending _not_ to do. A specific part made me confused:

> "Since alerts are not required to receive focus, content authors SHOULD NOT require users to close an alert."

Does this mean that it's actually okay to auto-hide it, given it doesn't require focus, therefore not requiring user action? With that said, I'd be ok not to add anything now, but it made me question if this recommendation is even correct (thus, worth adding a demo/section).

Reply:
I did some more reading on the Snackbar / Toast pattern, and it seems that because they're not intended to convey critical information - more just to confirm that an expected action has taken place (email sent, form submitted) - they should automatically be dismissed with no action taken by the user. MD2 suggests 4 to 10 seconds: https://m2.material.io/components/snackbars

So I do think we should add a section to demonstrate the `autoHideDuration` prop, and maybe move the "provide sufficient time" link to that section and cut the link about how it shouldn't auto-hide. What do you think?